spec/gush/workflow_spec.rb

```diff
@@ -121,6 +140,13 @@ def configure(*args)
       expect(flow.jobs.first.params).to eq ({ something: 1 })
     end
 
+    it "merges globals with params and passes them to the job" do
```

Comment:
maybe include here that the job param takes precedence over the global param (overwriting it in the job)